libs/dart/seed_cryptography/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: affinidi_tdk_seed_cryptography
+description: Cryptography package for Affinidi Vault's wallet seed
+version: 1.0.0
+repository: https://github.com/affinidi/affinidi-tdk
+
+environment:
+  sdk: ^3.6.0
+
+resolution: workspace
+
+dependencies:
+  affinidi_tdk_cryptography: ^1.0.0
```

Comment:
Cryptography package has a breaking change. May I ask to split this PR in two, one for the cryptography package with the breaking change and another one for the new seed encryption package with a pre-release?